Improve regex documentation (#4)
IDML/idml-docs: src/main/tut/functions/regex.md

section: language
---

> These functions allow you to manipulate strings using Java regular expressions. The underlying implementation is re2 and the Syntax is available at https://github.com/google/re2/wiki/Syntax.

Reviewer: ```suggestion
These functions allow you to manipulate strings using Java regular expressions. The underlying implementation is re2 (see [syntax](https://github.com/google/re2/wiki/Syntax)).
```
